### PR TITLE
SAAS-237 Add API endpoint

### PR DESCRIPTION
**Pull request by palash25**
https://jira.percona.com/browse/SAAS-237

### CODE REVIEW

**Comment by it-percona** [comment]
[CLA assistant check](https://cla-assistant.percona.com:443/percona/pmm?pullRequest=560) <br/>Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla-assistant.percona.com:443/percona/pmm?pullRequest=560) before we can accept your contribution.<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.percona.com:443/check/percona/pmm?pullRequest=560) it.</sub>

File: api/managementpb/checks.proto [hunk 1]
```diff
@@ -10,8 +10,28 @@ message StartSecurityChecksRequest {}
 
 message StartSecurityChecksResponse {}
 
+message GetSecurityCheckResultsRequest {}
+
+// STTCheckResult represents the check result returned from pmm-managed after running the check.
+message STTCheckResult {
+  string summary = 1;
+  string description = 2;
+  int32 severity = 3;
```
Comment (AlekSi) [inline api/managementpb/checks.proto:19]: Let's use enum there

File: api/managementpb/checks.proto [hunk 2]
```diff
@@ -10,8 +10,28 @@ message StartSecurityChecksRequest {}
...
+message GetSecurityCheckResultsResponse {
+  repeated STTCheckResult results = 1;
+}
+
 // SecurityChecks service provides public Management API methods for Security Checks Service.
 service SecurityChecks {
+  // GetSecurityCheckResults start Security Thread Tool checks.
+  rpc GetSecurityCheckResults(GetSecurityCheckResultsRequest) returns (GetSecurityCheckResultsResponse) {
+    option (google.api.http) = {
+      get: "/v1/management/SecurityChecks/GetCheckResults"
```
Comment (AlekSi) [inline api/managementpb/checks.proto:32]: All our APIs use POST. I will write down somewhere why tomorrow.

File: api/managementpb/checks.proto [hunk 3]
```diff
@@ -10,8 +10,42 @@ message StartSecurityChecksRequest {}
...
+  SEVERITY_CRITICAL = 3;
+  SEVERITY_ERROR = 4;
+  SEVERITY_WARNING = 5;
+  SEVERITY_NOTICE = 6;
+  SEVERITY_INFO = 7;
+  SEVERITY_DEBUG = 8;
+}
+
+// STTCheckResult represents the check result returned from pmm-managed after running the check.
+message STTCheckResult {
```
Comment (artemgavrilov) [inline api/managementpb/checks.proto:29]: I think better to name it `SecurityCheckResult` to be consistent with `GetSecurityCheckResultsRequest`, `GetSecurityCheckResultsResponse`.

Comment (BupycHuk) [reply api/managementpb/checks.proto:29]: +1

**Pull request merged**